### PR TITLE
Create OCM role with managed policies

### PR DESCRIPTION
**Pull request by oriAdler**
Attach existing policy instead of creating a new one.

Related: [SDA-7864](https://issues.redhat.com/browse/SDA-7864)

image

### CODE REVIEW

**Comment by oriAdler** [comment]
@zgalor @gdbranco @hunterkepley PTAL

File: cmd/create/ocmrole/cmd.go [hunk 1]
```diff
@@ -326,52 +356,95 @@ func buildCommands(prefix string, roleName string, rolePath string, permissionsB
...
+				AddParam(awscb.PolicyName, policyName).
+				AddParam(awscb.PolicyDocument, fmt.Sprintf("file://sts_%s_permission_policy.json", aws.OCMAdminRolePolicyFile)).
+				AddTags(adminTags).
+				AddParam(awscb.Path, rolePath).
+				Build()
+		}
 
+		if managedPolicies {
+			policyARN, err = aws.GetManagedPolicyARN(policies,
+				fmt.Sprintf("sts_%s_permission_policy", aws.OCMRolePolicyFile))
```
Comment (gdbranco) [inline cmd/create/ocmrole/cmd.go:407]: Is there an OCMAdmin Policy for managed policies?

File: cmd/create/ocmrole/cmd.go [hunk 2]
```diff
@@ -326,52 +356,95 @@ func buildCommands(prefix string, roleName string, rolePath string, permissionsB
...
+			if err != nil {
+				return "", err
+			}
+		} else {
+			policyARN = aws.GetPolicyARN(accountID, roleName, rolePath)
```
Comment (gdbranco) [inline cmd/create/ocmrole/cmd.go:412]: GetAdminPolicyARN

File: cmd/create/ocmrole/cmd.go (hunk 1 above)
Comment (oriAdler) [reply cmd/create/ocmrole/cmd.go]: Good catch!
Fixed, thanks.

File: cmd/create/ocmrole/cmd.go (hunk 2 above)
Comment (oriAdler) [reply cmd/create/ocmrole/cmd.go]: Missed that, fixed it.
Thanks.

**Pull request merged**